src/organisms/cards/CompanyCard/index.js

```diff
@@ -46,7 +47,7 @@ const CompanyCard = (props) => {
       className={classes}
     >
       <div>
-        <img className={styles.image} src={imageAttr.src} role='presentation' />
+        <img className={styles.image} src={imageAttr.src} alt='' />
```

Comment:
why `role` removed?

Reply:
(same as above)

the lint error was

```
Prefer alt="" over a presentational role. First rule of aria is to 
not use aria if it can be achieved via native HTML  
jsx-a11y/alt-text
```
so I replaced `role='presentation` with `alt=''`. But I just checked and it works if I keep both so I can add it back in if you'd prefer that.

Reply:
discussed offline - keeping `alt=''`